.github/ISSUE_TEMPLATE/launch.md

```diff
@@ -11,8 +11,8 @@ Please check off line-items as they are completed and leave notes if necessary.
 If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
 (e.g. `~~Not relevant item~~`) or remove it. If child tickets are created for
 any line-item, please update this ticket description to include references to
-those tickets.
 
+those tickets.
```

Comment:
This this intentional? In any case, whether it be this or my suggestion on `launch-analytics.md`, we should make the update on all of these.

Reply:
@adamzimmermann Unresolving this because I think the main issue I wanted to point out was not clear. This diff appears to move the `those tickets.` line one line down. Is that intentional? Also, should this template have the same intro text as the rest? And does `launch-initial.md` replace this template?

Reply:
Can you check the latest diff state. `those tickets` isn't in it at all anymore. 

> Also, should this template have the same intro text as the rest? And does launch-initial.md replace this template?

I'm sorting that out in the other comment thread, as you are correct in calling this out.

Reply:
I'm closing this for now, but please re-open or add a new comment if you think this is still an issue.